### DISCUSSION TITLE
deploying-asp-net-core-applications-to-kubernetes-part-2-configuring-resources-with-yaml-manifests/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#115] [Announcements]
# deploying-asp-net-core-applications-to-kubernetes-part-2-configuring-resources-with-yaml-manifests/

In this post I walk through the configuration manifests for the main resources app developers should understand when deploying to Kubernetes.

https://andrewlock.net/deploying-asp-net-core-applications-to-kubernetes-part-2-configuring-resources-with-yaml-manifests/

[a comment by an automated account is not shown]

**Comment by akhanalcs** [comment]
You have made Kubernetes very easy! Thank you so much! You're amazing!
